groups/sig-release/groups.yaml

```diff
@@ -215,14 +215,17 @@ groups:
     managers:
       - tpepper@vmware.com # Release Team subproject owner
     members:
-      - keroden@microsoft.com
-      - killen.bob@gmail.com
```

Comment:
Wondering why some of these were deleted?

Reply:
Per the description of the release-comms section "      External Release Communications list.
      Should be used when coordinating external release timelines e.g., with the CNCF.
      Membership should include SIG Chairs, Release Team Leads and Lead Shadows, and
      the Communications team for the current release cycle". They didn't fit it into any of the above (to the best of my knowledge), which is why I deleted them.